<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneEditor</class>
 <widget class="QDialog" name="PlaneEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plane Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0" columnstretch="0,1">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Plane</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="1">
       <widget class="QSpinBox" name="engine_critical_hp"/>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Max Altitude:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="tail_hp"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Fuel:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Wing HP:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QSpinBox" name="rated_dive"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Fuselage Critical HP:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="fuel_amt"/>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="engine_hp"/>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Rated Dive:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTabWidget" name="crew_editor_tab">
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QSpinBox" name="rated_climb"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Rated Climb:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Engine HP:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="fuselage_critical_hp"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Tail HP:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Stability Rating:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="wing_hp"/>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="fuselage_hp"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="plane_name">
        <property name="placeholderText">
         <string>Plane Name</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="stab_rating"/>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="wing_critical_hp"/>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="max_alt"/>
      </item>
      <item row="16" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Maneuver</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0">
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Level Value:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="set_dive">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="set_obs_can_reload">
           <property name="text">
            <string>Observer can reload</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="maneuver_selection"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="set_climb">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2" rowspan="9">
          <widget class="QGraphicsView" name="maneuver_preview_gv">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="interactive">
            <bool>false</bool>
           </property>
           <property name="resizeAnchor">
            <enum>QGraphicsView::AnchorViewCenter</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Climb Value:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Dive Value:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QCheckBox" name="set_weight_restricted">
           <property name="text">
            <string>Weight restricted</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="add_maneuver">
             <property name="text">
              <string>Add Maneuver</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>C:/Users/Wesley/Downloads/icons8-add-48.png</normaloff>C:/Users/Wesley/Downloads/icons8-add-48.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_maneuver">
             <property name="text">
              <string>Remove Maneuver</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>C:/Users/Wesley/Downloads/icons8-minus-48.png</normaloff>C:/Users/Wesley/Downloads/icons8-minus-48.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="set_level">
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="set_can_reload">
           <property name="text">
            <string>Can reload</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="set_can_put_out_fires">
           <property name="text">
            <string>Can put out fires</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QCheckBox" name="set_can_repeat">
           <property name="text">
            <string>Can be repeated</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="early_war_btn">
          <property name="text">
           <string>Early War</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">era_btn_grp</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="late_war_btn">
          <property name="text">
           <string>Late War</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">era_btn_grp</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Fuselage HP:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Engine Critical HP:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Wing Critical HP:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Tail Critical HP:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="tail_critical_hp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="export_json">
     <property name="text">
      <string>Export to JSON</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGraphicsView" name="maneuver_schedule_gv"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="era_btn_grp"/>
 </buttongroups>
</ui>
